client/src/components/CreatePost.tsx

```diff
@@ -36,147 +36,148 @@ function CreatePost() {
     let url = URL.createObjectURL((target.files as FileList)[0]);
-    setFile(url);
-    console.log(url);
+    setForm({ ...form, file: url });
   };
 
   return (
-    <ThemeProvider theme={theme}>
-      <Container component="main" maxWidth="md">
-        <CssBaseline />
-        <Box
-          sx={{
-            marginTop: 8,
-            display: "flex",
-            flexDirection: "column",
-            alignItems: "center",
-          }}
-        >
-          <Typography component="h3" variant="h3" fontWeight="bold">
-            Create Post
-          </Typography>
-          {/* form  begins*/}
-          <Box component="form" noValidate sx={{ mt: 2 }}>
-            <Grid container spacing={2}>
-              <Grid item xs={12}>
-                <TextField
-                  required
-                  fullWidth
-                  label="Title"
-                  placeholder="title"
-                  size="small"
-                  data-testid="titleTextField"
-                  onChange={(e) => setTitle(e.target.value)}
-                />
-              </Grid>
-
-              <Grid item xs={12}>
-                <TextField
-                  required
-                  fullWidth
-                  label="Body"
-                  data-testid="bodyTextField"
-                  placeholder="description"
-                  multiline
-                  rows={6}
-                  onChange={(e) => setBody(e.target.value)}
-                />
-              </Grid>
-
-              <Grid item xs={12} md={4}>
-                <TextField
-                  required
-                  label="Upload Thumbnail"
-                  type="file"
-                  fullWidth
-                  data-testid="fileField"
-                  InputLabelProps={{ shrink: true }}
-                  size="small"
-                  onChange={(e) => handleImageUpload(e)}
-                />
-              </Grid>
-
-              <Grid item xs={12} md={3}>
-                <TextField
-                  label="Event Capacity"
-                  placeholder="40"
-                  fullWidth
-                  size="small"
-                  onChange={(e) => setEventCapacity(e.target.value)}
-                />
-              </Grid>
-
-              <Grid item xs={12} md={5}>
-                <TextField
-                  fullWidth
-                  label="Location"
-                  placeholder="Deerfield Hall"
-                  size="small"
-                  onChange={(e) => setEventLocation(e.target.value)}
-                />
-              </Grid>
-
-              <Grid item xs={12}>
-                <TextField
-                  fullWidth
-                  label="Tags (Separate tags by a comma)"
-                  placeholder="Clubs, Math, MCS"
-                  size="small"
-                  onChange={(e) => setTags(e.target.value)}
-                />
-              </Grid>
+    <Container component="main" maxWidth="md">
+      <CssBaseline />
```

Comment:
Do we need this here?

Reply:
Are you referring to <CssBaseline/>?

Reply:
I don't think we need <Cssbaseline>, check to make sure